variables.tf

```diff
@@ -234,27 +248,50 @@ variable "repo_config_file" {
 ## Rest of Azure Container Group Configs ##
 ###########################################
 
-variable "environment_variables" {
-  description = "A list of environment variables to be set on the container"
-  type        = map(string)
-  default     = {}
-}
-
-variable "secure_environment_variables" {
-  description = "A list of sensitive environment variables to be set on the container"
-  type        = map(string)
-  default     = {}
-}
-
-variable "secure_environment_variables_from_key_vault" {
-  description = "A list of sensitive environment variables to be set on the container read from Azure Key Vault"
+variable "containers" {
```

Comment:
What's the difference between that variable and `atlantis_container`?

Reply:
it gives you flexibility to add your own sidecar containers should you want them deploy along the atlantis one